### PR TITLE
Release v1.2.0

### PR DESCRIPTION
**Pull request by msm-code**
**Your checklist for this pull request**
- [x] I've read the [contributing guideline](https://github.com/CERT-Polska/mquery/blob/master/CONTRIBUTING.md).
- [n/a] I've tested my changes by building and running mquery, and testing changed functionality (if applicable)
- [n/a] I've added automated tests for my change (if applicable, optional)
- [n/a] I've updated documentation to reflect my change (if applicable)

**What is the current behaviour?**
The version is 1.1.0

**What is the new behaviour?**
The version is 1.2.0

**Test plan**
Celebrate :tada:

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Version 1.2
+
+### UI Improvements
+
+A lot of UI improvements
```
Comment (ITAYC0HEN) [inline RELEASES.md:5]: Please be more detailed, there are many changes. Improved progress bar, monaco editor, yara code completion, better status page, etc...

This snippet can be helpful to pick the ones relevant.
```
git log --pretty=oneline v1.1..HEAD
```

File: RELEASES.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+![](./docs/interface-v1.2.gif)
+
+### New features
```
Comment (ITAYC0HEN) [inline RELEASES.md:9]: I would put it above the UI.

You can also go for a top 5 chnages, and then a detailed list splitted to categories )new features, buf fixes, performance, ui, ...)

File: RELEASES.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- Result streaming (improved time to the first result) [(#59)](https://github.com/CERT-Polska/mquery/pull/59).
```
Comment (ITAYC0HEN) [inline RELEASES.md:11]:
```suggestion
- Results streaming (first results appear faster) [(#59)](https://github.com/CERT-Polska/mquery/pull/59)
```

File: RELEASES.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- Support for distributed Ursadb in the backend. [(#119)](https://github.com/CERT-Polska/mquery/pull/119).
```
Comment (ITAYC0HEN) [inline RELEASES.md:12]: Double dots
```suggestion
- Support for distributed Ursadb in the backend [(#119)](https://github.com/CERT-Polska/mquery/pull/119)
```

File: RELEASES.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+- Powerful plugin suppor (See the [./docs/plugins.md](documentation))
```
Comment (ITAYC0HEN) [inline RELEASES.md:13]: typo
```suggestion
- Powerful plugin support (See the [./docs/plugins.md](documentation))
```

File: RELEASES.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+- Support for ursadb tags (for example, to tag collections as benign or malicious) [(#44)](https://github.com/CERT-Polska/mquery/pull/44)
+
+### Improvements
```
Comment (ITAYC0HEN) [inline RELEASES.md:16]: as discussed seperately, remove the dots

File: RELEASES.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    - Regexes [(#169)](https://github.com/CERT-Polska/mquery/pull/169).
+    - Strings that are both `ascii` and `wide` [(#65)](https://github.com/CERT-Polska/mquery/pull/65).
+    - `xor` modifier (without ranges) [(#98)](https://github.com/CERT-Polska/mquery/pull/98).
+    - Anonymous variables [(#66)](https://github.com/CERT-Polska/mquery/pull/66).
+- Results view improvements [issue #82](https://github.com/CERT-Polska/mquery/issues/82).
+
+### Others
+
+- Much better workflow for new contributors [(#47)](https://github.com/CERT-Polska/mquery/pull/47).
+- A lot of [documentation](https://cert-polska.github.io/mquery/).
```
Comment (ITAYC0HEN) [inline RELEASES.md:31]:
```suggestion
- Improved [documentation](https://cert-polska.github.io/mquery/).
```
or
```suggestion
- New [documentation](https://cert-polska.github.io/mquery/).
```

**Comment by msm-code** [comment]
Applied suggestions

`Also, the commit is not tagged v1.2.0`

Shouldn't I do this after the merge?

File: RELEASES.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
+# Version 1.2
+
+![](./docs/interface-v1.2.gif)
+
+### New features
+
+- Results streaming (first results appear faster) [(#59)](https://github.com/CERT-Polska/mquery/pull/59)
+- Support for distributed Ursadb in the backend [(#119)](https://github.com/CERT-Polska/mquery/pull/119)
+- Powerful plugin support (See the [./docs/plugins.md](documentation))
+    - With `/config` endpoint, to configure them using the web UI [(#132)](https://github.com/CERT-Polska/mquery/pull/132)
```
Comment (ITAYC0HEN) [inline RELEASES.md:10]:
```suggestion
    - Configurable from the web UI [(#132)](https://github.com/CERT-Polska/mquery/pull/132)
```

File: RELEASES.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+- Support for ursadb tags (for example, to tag collections as benign or malicious) [(#44)](https://github.com/CERT-Polska/mquery/pull/44)
```
Comment (ITAYC0HEN) [inline RELEASES.md:11]:
```suggestion
- Support for Ursadb tags (for example, to tag collections as benign or malicious) [(#44)](https://github.com/CERT-Polska/mquery/pull/44)
```

File: RELEASES.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+    - Download matched files or sha256 hashes [(#176)](https://github.com/CERT-Polska/mquery/pull/#176) [(#163)](https://github.com/CERT-Polska/mquery/pull/#163)
+    - Add pagination to the results table [(#96)](https://github.com/CERT-Polska/mquery/pull/#96)
+- Status page improvements:
+    - Display file count along with dataset size [(#91)](https://github.com/CERT-Polska/mquery/pull/#91)
+- Filter jobs by author, status and others [(#152)](https://github.com/CERT-Polska/mquery/pull/#152)
+- Show number of errors (for example, missing files) per job [(#148)](https://github.com/CERT-Polska/mquery/pull/#148)
+
+### Improvements
+
+- Much better Yara support [issue #41](https://github.com/CERT-Polska/mquery/issues/41):
```
Comment (ITAYC0HEN) [inline RELEASES.md:29]:
```suggestion
- Much better Yara support [(issue #41)](https://github.com/CERT-Polska/mquery/issues/41):
```

File: RELEASES.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+    - Multiple rules in a query [(#55)](https://github.com/CERT-Polska/mquery/pull/55)
+    - Private and global rules [(#55)](https://github.com/CERT-Polska/mquery/pull/55)
+    - Case insensitive strings (`nocase` modifier) [(#136)](https://github.com/CERT-Polska/mquery/pull/136)
+    - Regexes [(#169)](https://github.com/CERT-Polska/mquery/pull/169)
+    - Strings that are both `ascii` and `wide` [(#65)](https://github.com/CERT-Polska/mquery/pull/65)
+    - `xor` modifier (without ranges) [(#98)](https://github.com/CERT-Polska/mquery/pull/98)
+    - Anonymous variables [(#66)](https://github.com/CERT-Polska/mquery/pull/66)
+- Change the web server used to FastAPI (to improve performance and get API documentation for free)
```
Comment (ITAYC0HEN) [inline RELEASES.md:37]: what do you mean by "used"? maybe "usage"?
or maybe "Use FastAPI to ..."

File: RELEASES.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+    - Multiple rules in a query [(#55)](https://github.com/CERT-Polska/mquery/pull/55)
+    - Private and global rules [(#55)](https://github.com/CERT-Polska/mquery/pull/55)
+    - Case insensitive strings (`nocase` modifier) [(#136)](https://github.com/CERT-Polska/mquery/pull/136)
+    - Regexes [(#169)](https://github.com/CERT-Polska/mquery/pull/169)
+    - Strings that are both `ascii` and `wide` [(#65)](https://github.com/CERT-Polska/mquery/pull/65)
+    - `xor` modifier (without ranges) [(#98)](https://github.com/CERT-Polska/mquery/pull/98)
+    - Anonymous variables [(#66)](https://github.com/CERT-Polska/mquery/pull/66)
+- Change the web server used to FastAPI (to improve performance and get API documentation for free) 
+- Document the API and add swagger UI to `/docs` endpoint
+- Cache parsed Yara rules.
```
Comment (ITAYC0HEN) [inline RELEASES.md:39]:
```suggestion
- Cache parsed Yara rules
```

File: RELEASES.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+- Batch files when matching yara rules, to improve performance
+
+### Others
+
+- Much better workflow for new contributors [(#47)](https://github.com/CERT-Polska/mquery/pull/47)
+- Varions utility scripts [(#134)](https://github.com/CERT-Polska/mquery/pull/134/), including
```
Comment (ITAYC0HEN) [inline RELEASES.md:45]:
```suggestion
- Various utility scripts [(#134)](https://github.com/CERT-Polska/mquery/pull/134/), including
```

**Pull request merged**